Sources/SnowManLogicSpecs/ViewSpec.swift

```diff
@@ -0,0 +1,122 @@
+import SnowManLogic
+import Swiftest
+
+class ViewSpec: Swiftest.Spec {
+  	let spec = describe("View logic") {
+		var io: MockIO!
+        	var view: View!
+	        before() {
+			io = MockIO()
+       		 	view = View(io: io)
+        	}
+        
+       		describe("#receiveGuess") {
+        		it("should return the user input if it is a string of letters") {
+            		io.stubbedUserInput = ["hello"]
+                    		expect(view.receiveGuess()).to.equal("hello")
+           		}
+
+           		it("should return the user input if it is a single letter") {
+                		io.stubbedUserInput = ["a"]
+                		expect(view.receiveGuess()).to.equal("a")
+            		}
+
+            		it("should return an error message if the user input is a number") {   
+                		io.stubbedUserInput = ["1", "fail"]
+                		expect(view.receiveGuess()).to.equal("fail")
+            		}
+
+                	it("should return an error message if the user input is a non-letter character") {
+                        	io.stubbedUserInput = ["?", "fail"]
+                        	expect(view.receiveGuess()).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user input is a combination of valid and invalid inputs") {
+                        	io.stubbedUserInput = ["?a", "fail"]
+                        	expect(view.receiveGuess()).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user hits return") {
+                        	io.stubbedUserInput = ["", "fail"]
+                        	expect(view.receiveGuess()).to.equal("fail")
+                    	}
+        	}
+
+        	describe("#assignBlanks") {
+            		var gameWord: String!
+            		var correctGuesses: [String]!
+            
+            		before() {
+                		gameWord = "hello"
+                		correctGuesses = []
+            		}
+            
+            		it("should return only blanks for unguessed word") {
+	                        io.stubPrintedOutput("__  __  __  __  __")
```

Comment:
Not really sure if this is the right approach because I am not actually checking what the program is going to return. This is the part about testing that confuses me, when there is no return value to test.